plugins/providers/virtualbox/driver/version_5_0.rb

```diff
@@ -923,6 +929,32 @@ def vdi_to_vmdk(defined_disk_path)
           destination
         end
 
+        # Helper method to get a list of storage controllers added to the
+        # current VM
+        #
+        # @return [Array<VagrantPlugins::ProviderVirtualBox::Model::StorageController>]
+        def read_storage_controllers
+          vm_info = show_vm_info
+          count = vm_info.count { |key, value| key.match(/^storagecontrollername\d+$/) }
+
+          (0..count - 1).map do |n|
```

Comment:
Hmm, at first glance it seems like this should be a `.each` instead of a `.map`. We're just iterating over a sequence `n` here I think?

Reply:
Since the result of the map is what is being returned from the method, it should be `#map` as that's building the `Array` result :smiley: